recipes/migrate-n/run_test.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+if os.system('migrate-n -h') != 65280:
```

Comment:
can you migrate this into the yaml file and do something like this `migrate-n -h | grep "Foo"` ... with Foo a text included in the help message. This way the error code is taken from the grep step and it should be 0.